docs/topics/api.rst

```diff
@@ -91,17 +91,25 @@ Sign-In
 ~~~~~~~
 
 Authenticates the user and saves state to a session cookie (delivered in the
-response).
+response). You can sign in either with an `authorization code`_ or an
+`access token`_, as explained below. If you sign in with an
+`authorization code`_, it will be exchanged for an `access token`_
+which will then be verified.
+The access token derived from any sign-in method must be scoped for
+**payments** and **profile:email** otherwise the sign-in will fail.
 
 .. http:post:: /api/auth/sign-in/
 
     **Request**
 
+    :param string authorization_code:
+        An internally created `Firefox Accounts`_ OAuth `authorization code`_.
+        This would typically be generated using an internal client ID from
+        a login screen on the payments management app.
```

Comment:
This allows us to present a login screen on the management page instead of accepting a token that was generated from a login screen on Mozilla Concrete.